core/rootcling_stage1/CMakeLists.txt

```diff
@@ -17,7 +17,9 @@ else()
   endif()
 endif()
 
-if(NOT builtin_clang)
+if(builtin_clang)
```

Comment:
This condition is intended to pin `clingMetaProcessor` as a library when we use `builtin_clang=Off`. That is, when clang is externally prebuilt and we do not really need to track dependencies. I believe this change will lead to missing dependencies and we will start seeing compilation errors. Did you test that code with compiling ROOT with `-Dbuiltin_clang=Off`?

Reply:
Hi @vgvassilev , thanks for the comment. Isn't the case of "-D builtin_clang=Off" honoured by the second branch of this if statement, which is unchanged with respect to before?

Reply:
Ah, yes, you are right, misread the diff, apologies. I suspect that linking `clingMetaProcessor` for the `-Dbuiltin_clang=On` is enough as the llvm build system will expand automatically its dependencies. Maybe worth a comment since I am not sure if that's a standard cmake behavior.

Reply:
It's standard cmake behavior we use throughout ROOT.